docs/reference/requirement-sets/excel-api-online-requirement-set.md

```diff
@@ -37,7 +37,23 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
 
 | Class | Fields | Description |
 |:---|:---|:---|
-|[Range](/javascript/api/excel/excel.range)|[getMergedAreas()](/javascript/api/excel/excel.range#getmergedareas--)|Returns a RangeAreas object that represents the merged areas in this range.|
+|[NamedSheetView](/javascript/api/excel/excel.namedsheetview)|[activate()](/javascript/api/excel/excel.namedsheetview#activate--)|Activates this sheet view.|
```

Comment:
@ElizabethSamuel-MSFT Where are we on the link anchor issue with OPS?

Reply:
Current estimate is this quarter